src/PointwiseFunctions/AnalyticSolutions/EinsteinSolutions/Minkowski.cpp

```diff
@@ -103,6 +103,22 @@ tnsr::ii<T, Dim> Minkowski<Dim>::extrinsic_curvature(const tnsr::I<T, Dim> &x,
 noexcept {
   return tnsr::ii<T, Dim>(make_with_value<T>(x, 0.));
 }
+
+template<size_t Dim>
+template<typename T>
+tnsr::Ijj<T, Dim> Minkowski<Dim>::christoffel_2nd_kind(const tnsr::I<T, Dim>& x,
```

Comment:
We should talk about this in a hangout sometime.  I don't think we want Minkowski, KerrSchild, etc to have functions christoffel_2nd_kind and so on. Instead they should have only the basic functions (metric and derivatives), and the PointwiseFunctions should be used to compute everything else.

Reply:
The problem is we need some analytic functions like this to compute curvature analytically, without setting up a mesh and taking numerical derivatives.

Reply:
I told Geoffrey to add them to KerrSchild as they are very useful for testing.  There is no reason we need to add them to every Einstein solution.

Reply:
I like the idea of including them in KerrSchikd and in Minkowski, because I plan to test horizon curvature of both. But I can hard code the zeros in the flat space test if you feel strongly I should remove the Christoffel functions from Minkowski.

Reply:
I removed the code and test for Minkowski to compute christoffel_2nd_kind and its derivative.